CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Added
+
+- Single-transaction historical query support has been added to JavaScript endpoints (#2285).
+
```

Comment:
Does single-transaction mean point queries as opposed to range queries?

Reply:
Yes. I'll change it.